arches/app/views/api.py

```diff
@@ -1609,3 +1609,15 @@ def get(self, request):
             return JSONResponse(str(e), status=500)
 
         return JSONResponse({"data": result})
+    
+class GetNodegroupTree(APIBase):
+    """
+    Returns the path to a nodegroup from the root node. Transforms node alias to node name.
+    """
+    def get(self,request):
```

Comment:
Re: permissions, should we check user role or permitted nodegroups?

Reply:
Good point @jacobtylerwalls. We probably should call 
```
from arches.app.utils.permission_backend import get_nodegroups_by_perm
[str(nodegroup.pk) for nodegroup in get_nodegroups_by_perm(request.user, "models.read_nodegroup")]
```
 and then filter `result` on 1621 by those nodegroupids.